app/js/app/app.js

```diff
@@ -89,7 +89,8 @@ define([
 
         // Here we configure the api provider with the server running the API. Don't need to do this if we want to use the same server as the static content.
         if (document.location.hostname == "localhost") {
-            apiProvider.urlPrefix("http://localhost:8080/isaac-api/api");
+            apiProvider.urlPrefix("https://staging.isaacphysics.org/api/any/api");
```

Comment:
These lines will need to be returned back to what they were, and you can just have the changes locally without committing them.